src/libsyntax/parse/parser.rs

```diff
@@ -387,6 +401,11 @@ impl<'a> Parser<'a> {
         parser
     }
 
+    pub fn with_interp_user_fn(&mut self, interp_user_fn: Option<Lrc<InterpUserFn>>) -> &mut Self {
```

Comment:
A method named `with_` typically takes a function.

Reply:
It does... kind of. But yeah, `set_interp_user_fn` makes a bit more sense, so I've renamed to that.